inst/qml/qml_components/RobustBayesianMetaAnalysisAdvanced.qml

```diff
@@ -0,0 +1,244 @@
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+import QtQuick 2.8
+import QtQuick.Layouts 1.3
+import JASP.Controls 1.0
+import JASP.Widgets 1.0
+
+Section
+{
+	property string analysisType: "RoBMA"
+	// RoBMA: Robust Bayesian Meta-Analsis
+	// BiBMA: Binomial Bayesian Meta-Analysis
+	// NoBMA: Normal Bayesian Meta-Analysis
+
+	property bool measuresGeneralChecked:	false
+	property bool measuresFittedChecked:		false
+
+	title: 				qsTr("Advanced")
+	columns: 			2
+	enabled:			!measuresFittedChecked
+	onEnabledChanged:	if(!enabled) expanded = false
+	
+	Group
+	{
+		rowSpacing: 10 * preferencesModel.uiScale
+
+		DropDown
+		{
+			name:		"advancedEstimationScale"
+			label:		qsTr("Estimation scale")
+			visible:	!measuresGeneralChecked && analysisType === "RoBMA"
+			values: [
+				{ label: qsTr("Fisher's z"),		value: "fishersZ"},
+				{ label: qsTr("Cohen's d"),			value: "cohensD"},
+				{ label: qsTr("logOR"),				value: "logOr"}
+			]
+		}
+
+		Group
+		{
+			title: 		qsTr("MCMC")
+
+			IntegerField
+			{
+				name:			"advancedMcmcAdaptation"
+				label:			qsTr("Adaptation")
+				defaultValue:	500
+				min:			100
+				fieldWidth:		55 * preferencesModel.uiScale
+			}
+			IntegerField
+			{
+				name:			"advancedMcmcBurnin"
+				label:			qsTr("Burnin")
+				defaultValue:	2000
+				min:			100
+				fieldWidth:		55 * preferencesModel.uiScale
+			}
+			IntegerField
+			{
+				name:			"advancedMcmcSamples"
+				label:			qsTr("Samples")
+				defaultValue:	5000
+				min:			100
+				fieldWidth:		55 * preferencesModel.uiScale
+			}
+			IntegerField
+			{
+				name:			"advancedMcmcChains"
+				label:			qsTr("Chains")
+				defaultValue:	3
+				min:			1
+				fieldWidth:		55 * preferencesModel.uiScale
+			}
+			IntegerField
+			{
+				name:			"advancedMcmcThin"
+				label:			qsTr("Thin")
+				defaultValue:	1
+				min:			1
+				fieldWidth:		55 * preferencesModel.uiScale
+			}
+
+		}
+
+		SetSeed{}
+	}
+
+	
+	Group
+	{
+
+		CheckBox
+		{
+			label:			qsTr("Autofit")
+			name:			"autofit"
+			checked:		true
+
+			CheckBox
+			{
+				label: 				qsTr("R-hat")
+				name:				"advancedAutofitRHat"
+				checked:			true
+				childrenOnSameRow:	true
+
+				DoubleField
+				{
+					name:			"advancedAutofitRHatTarget"
+					defaultValue:	1.05
+					min:			1
+					inclusive:		JASP.None
+				}
+			}
+
+			CheckBox
+			{
+				label: 				qsTr("Effective sample size")
+				name:				"advancedAutofitEss"
+				checked:			true
+				childrenOnSameRow:	true
+
+				DoubleField
+				{
+					name:			"advancedAutofitEssTarget"
+					defaultValue:	500
+					min:			1
+					inclusive:		JASP.None
+				}
```

Comment:
I wonder if it makes sense to have this depend on the number of samples?

Reply:
yeah, that would make sense, but I believe we don't do it at other places and I wouldn't like to have the autofit options change automatically once I adjust the rest of MCMC settings